tests/integration_tests/functional/test_mmds.py

```diff
@@ -116,6 +118,31 @@ def test_custom_ipv4(test_microvm_with_ssh, network_config):
     _, stdout, _ = ssh_connection.execute_command(cmd)
     assert json.load(stdout) == data_store['latest']['meta-data']
 
+@pytest.mark.timeout(1200)
+def test_big_buffer(test_microvm_with_ssh, network_config):
+    """
+    Test send large buffer.
+
+    @type: functional
+    """
+    test_microvm = test_microvm_with_ssh
+    test_microvm.spawn()
+
+    response = test_microvm.mmds.get()
+    assert test_microvm.api_session.is_status_ok(response.status_code)
+    assert response.json() == {}
+
+    data_store = {
+        'latest': {
+            'meta-data': {
+            }
+        }
+    }
+
+    chars = ''.join(['b' for i in range(190*1024*1024)])
```

Comment:
You don't necessarily need to go over the whole range. It's inefficient and takes about 5 seconds to do it - just tested this.

You can break it down in multiple loops:
```python
slice_1mb = list(itertools.repeat('x', 1024 * 1024))

slice_190mb = []
[slice_190mb.extend(slice_1mb) for _ in range(190))
```
This is like 10x faster.

We could also avoid creating a 190mb buffer in memory :)